backend/editor/api.py

```diff
@@ -242,4 +274,15 @@ async def editFooter(incomingData: Footer):
     convertedData = incomingData.dict()
     result = update_nodes("TEXT", "__footer__", convertedData)
     updatedFooter = list(result)
-    return updatedFooter
\ No newline at end of file
+    return updatedFooter
+
+# Delete methods
+
+@app.delete("/nodes")
+async def deleteNode(request: Request):

```

Comment:
BTW why didn't you choose to have:
```suggestion
@app.delete("/nodes/{id}")
async def deleteNode(request: Request):
```

Reply:
I wanted to keep the `/nodes` endpoint uniform. So, GET, POST, DELETE perform different functions and the request body only must be changed.

Reply:
Yes but delete is more like editEntry than list of nodes.

In REST when you use DELETE, it's on a url representing an entry.

Reply:
But, using `/entry/{entry}` would mean the deletion of an entry only. This endpoint would be able to delete any node in the taxonomy. So, I thought it should have the `/nodes` endpoint.